__tests__/api.test.js

```diff
@@ -95,10 +95,47 @@ describe('GET /api/articles/:article_id', () => {
       .get('/api/articles/609')
       .expect(404)
       .then(({ body })=> {
-        expect(body.msg).toBe('Not found')
+        expect(body.msg).toBe('Article not found')
       })
     });
   });
+
+describe('GET /api/articles', () => {
+    test('200: responds with an array of all aricle objects sorted by date in DESC order', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body })=> {
+            const { articles } = body;
+            
+            expect(articles).toHaveLength(13)
+
+            articles.forEach((article)=> {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: expect.any(String),
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(Number)
+                });
+            });
+
+            expect(articles).toBeSortedBy('created_at',{ descending: true, });
+        });
+    });
+
+    test('404: responds with "Route not found" if passed an invalid route', () => {
```

Comment:
Great 404 test - it doesn't belong to this endpoint though so I'd pop it in it's own describe block